internal/saucecloud/espresso.go

```diff
@@ -92,13 +80,64 @@ func (r *EspressoRunner) runSuites(appFileID string, testAppFileID string) bool
 	return r.collectResults(r.Project.Artifacts.Download, results, jobsCount)
 }
 
+// enumerateDevices returns a list of device targeted by the current suite.
+func enumerateDevices(d config.Device) []deviceConfig {
+	if d.ID != "" {
+		return []deviceConfig{{ID: d.ID, platformName: d.PlatformName}}
+	}
+	// FIXME: Implement RDC dynamic
```

Comment:
? 😄

Reply:
This is a reminder that we need to implement the dynamic device query. This will be achieved in another PR when specification will be validated 😄